File: Java/27. Remove Element.java
package Java;
class Solution {
    public int removeElement(int[] nums, int val) {

        int curr = 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==val) continue;
            nums[curr++] = nums[i];
        }
        return curr;
    }
}

File: Javascript/27. Remove Element.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var removeDuplicates = function(nums,val) {

    let curr = 0;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] === val) continue;
        nums[curr++] = nums[i];
    }
    return curr;
    
};